jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -621,6 +623,9 @@ def _update_stretch_histogram(self, msg={}):
             if len(sub_data) > 0:
                 hist_lims = interval.get_limits(sub_data)
                 hist_mark.min, hist_mark.max = hist_lims
+                stretch_vstep = (hist_lims[1] - hist_lims[0]) / 100.

```

Comment:
Maybe let's not hard code this. Should we make a kwarg/attr for the divisor here, so this can be adjusted?

Reply:
I'll add a comment explaining that the 100 is arbitrary and can be adjusted (we can either change it in the code or expose a traitlet to be user-customizable in the future if we ever want to, without being confused that it serves some specific purpose)